package.json

```diff
@@ -13,6 +13,10 @@
 	},
 	"type": "commonjs",
 	"main": "./common.js",
+	"files": [
+		"*.js",
+		"src/*.js"
```

Comment:
```suggestion
		"**/*.js"
```

Reply:
`yarn pack` doesn't catch all .js files in the root 
![image](https://user-images.githubusercontent.com/42935195/188483291-4356dfac-d991-4e79-90fd-b5d74268ed4f.png)